app/src/main/java/org/thoughtcrime/securesms/notifications/DefaultMessageNotifier.java

```diff
@@ -353,9 +351,17 @@ private void sendSingleThreadNotification(@NonNull  Context context,
     // TODO: attributes to perform the colour lookup. Also, it makes little sense to highlight the mentions using
     // TODO: the app theme as it may result in insufficient contrast with the notification background which will
     // TODO: be using the SYSTEM theme.
-    builder.setPrimaryMessageBody(recipient, notifications.get(0).getIndividualRecipient(),
-                                  //MentionUtilities.highlightMentions(text == null ? "" : text, notifications.get(0).getThreadId(), context), // Removing hightlighting mentions -ACL
-                                  text == null ? "" : text,
+    CharSequence builderCS = text == null ? "" : text;
+    SpannableString ss = MentionUtilities.highlightMentions(builderCS,
+            false,
+            false,
+            true,
+            bundled ? notifications.get(0).getThreadId() : 0,
+            context);
```

Comment:
It's nice to have that 1st param down there, so you don't miss it.

It's nice to close it on the next line analogous to closing `{}`s etc.

```suggestion
    SpannableString ss = MentionUtilities.highlightMentions(
            builderCS,
            false,
            false,
            true,
            bundled ? notifications.get(0).getThreadId() : 0,
            context
    );
```

Reply:
Good point - I think I like that way better, too.